src/pages/_app.tsx

```diff
@@ -37,6 +38,9 @@ function MyApp({Component, pageProps}: AppProps) {
     const [locale, setLocale] = useState<string | undefined>(undefined);
     const [theme, setTheme] = useState<ThemePrepared>(teamsTheme);
 
+    const router = useRouter();
```

Comment:
also add `?` here (optional chaining)

Reply:
Not needed, query is always present, and in case of botName the fallback is an empty string